drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -528,7 +538,9 @@ local function do_configure(driver, device)
     -- Add nobattery profiles if updated
```

Comment:
Just calling out this comment above, we are now handling the `no-battery` portion of this, but also we should expand on this reading of the attribute list to include `ThermostatRunningState` in a separate PR